hostprocess/flannel/kube-proxy/kube-proxy.yml

```diff
@@ -21,14 +21,16 @@ spec:
           runAsUserName: "NT AUTHORITY\\system"
       hostNetwork: true
       containers:
-      - image: sigwindowstools/kube-proxy:v1.22.4-flannel-hostprocess
+      - image: sigwindowstools/kube-proxy:v1.24.3-flannel-hostprocess
         name: kube-proxy
         imagePullPolicy: Always
         env:
         # kube-proxy will use this value to select the correct network
         # https://github.com/kubernetes/kubernetes/blob/b0bc8adbc2178e15872f9ef040355c51c45d04bb/pkg/proxy/winkernel/proxier.go#L155-L158
         - name: KUBE_NETWORK
           value: "flannel.4096"
+        - name: CNI_BIN_PATH
+          value: C:\opt\cni\bin
```

Comment:
this isn't the same as the flannel install path above (C:\\Program Files\\containerd\\cni\\bin)

Reply:
since the script uses the default containerd paths, it will need to be `C:\Program Files\containerd\cni\bin`

Reply:
I just noticed the Install-containerd.ps1 script does update the containerd config to `C:\opt\cni\bin`. It doesn't really matter what the folders are, but we do need to be consistent across these files.

Reply:
the script also downloads some cni binaries and sticks them into `C:\opt\cni\bin`. We don't need that anymore either as the CNI are set up to bring their own required binaries.

Reply:
updated

Reply:
I modified the path